### PR TITLE
Use lorem ipsum-like words for word methods

### PR DESCRIPTION
**Pull request by justinvdm**
⚠️ **note** This is a breaking change - the same input for copycat word methods (`word()`, `words()`, `sentence()`, `paragraph()`) will now result in different output.

## tl;dr

Changes `copycat.word()` and other word methods from generating text like this:

```
Kai ni viramira memayo kayu. Hahyceavi nameta mohy shichiacea menivayu shi mika yokinmu, nahyraki hyka chi niceavi ta. Ta hamevakin yuno hyakova nivami yohycea ko, yoha shiyu miha hy kiko kinyoshi ka ninoshi. Notakimu yo yukake kakekaihy vaceaso vakiso nomu rae, yukin chiraekimo ceavino yo muyo. Hyva memayo shikemavi ka kakesokin mamuhamo kinmukame mora, ranino masochiyo kinoa kesoni mamo. Va nohakin komiva shimo hykikayo makinra yorae, sovami kai raenira raeyo sonavi mo mora chirae
```

To text like this:

```
Et modo lucilias legatomnem et. Quis ratio iudicur ut defuitur quod interessar endis, doloria romandum athenisse explicem quia. Expeten quam hoc ex amus ant sive, providintem ad claudicur torquato nes nihil nec ut. Audiri dicerea summum arisset ne exceperem tam si, amartifex doloris nam quae ipsum. Et causa iudicitat extremum endam tota tum antippus, de vidi videbo rerum ut. Affere ab mundi nimium summa partemerror causae, his am semperfruique in sapiens gloriatur et dicenim
```
 
## Context
The idea behind the way we currently generate words, was to use Japanese-like syllables for simplicity, since they compose easily (any syllable can follow any other).

It would be more ideal though to use something more standard and expected, like "lorem ipsum" placeholder text. It is also arguably a better representation of fake text that is meant to be read (e.g. on a web page) - since we generate words with latin characters. Latin words are naturally more representative of where these characters would be seen. In contrast, the previous fake text is not - it is closer to resembling a romanized form of text from a language not written with latin characters, and so it is probably less representative of what would appear as text meant to be read (e.g. on a web page).

 ## Approach

*note* The actual change is in a fictional PR that (atow) still needs to be released: https://github.com/oftherivier/fictional/pull/35.

* Take an except of [Cicero's De finibus bonorum et malorum](https://en.wikipedia.org/wiki/De_finibus_bonorum_et_malorum) (where lorem ipsum placeholder text is typically taken from or based on)
* Parse the syllables in it + the probability of each syllable is following each other syllable [(script here)](https://github.com/oftherivier/fictional/pull/35/files#diff-3c55c003e49143de7b0082b548d67132b8d05cb59d249c346fe3b07e92ea9e92). Store this as a json file.
* Use this data to generate lorem ipsum-like words in a [Markov chain-like](https://en.wikipedia.org/wiki/Markov_chain) way: for each syllable, account for the probabilities given in the data to decide which syllable should follow

### CODE REVIEW

File: records/benchmarks/0.17.3--after-lorem.txt [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+        "marginOfError": 0.00037423355135918755,
+        "relativeMarginOfError": 1.8999704668476336,
+        "standardErrorOfMean": 0.00019093548538734058,
+        "sampleVariance": 0.0000024425760918666565
+      },
+      "completed": true,
+      "percentSlower": 0
+    },
+    {
+      "ms": 1000,
```
Comment (justinvdm) [inline records/benchmarks/0.17.3--after-lorem.txt:360]: At 100,000 words, performance is pretty much identical that it was before.

**Review by peterp — APPROVED** [review]

File: records/benchmarks/0.17.3--after-lorem.txt [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+        "marginOfError": 0.000013925618399242639,
+        "relativeMarginOfError": 0.7798541724828908,
+        "standardErrorOfMean": 0.000007104907346552367,
+        "sampleVariance": 4.846052006697002e-9
+      },
+      "completed": true,
+      "percentSlower": 0
+    },
+    {
+      "ms": 200,
```
Comment (justinvdm) [inline records/benchmarks/0.17.3--after-lorem.txt:261]: At 10,000 words, it takes `200ms` for `copycat.sentence()` over the previous `166ms` unfortunately. Generating sentences this long is not something that feels like a likely use case though. That, and if it were, both these numbers aren't great.

File: records/benchmarks/0.17.3--after-lorem.txt [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+        "marginOfError": 2.9640514205745927e-7,
+        "relativeMarginOfError": 0.2914457565479899,
+        "standardErrorOfMean": 1.512271132946221e-7,
+        "sampleVariance": 2.264094339747022e-12
+      },
+      "completed": true,
+      "percentSlower": 0
+    },
+    {
+      "ms": 17.24137931034483,
```
Comment (justinvdm) [inline records/benchmarks/0.17.3--after-lorem.txt:162]: At 1000 words, there isn't much difference - was previously around 16ms, now 17ms.

File: records/benchmarks/0.17.3--after-lorem.txt [hunk 4]
```diff
@@ -0,0 +1,396 @@
+Running "100 words" suite...
```
Comment (justinvdm) [inline records/benchmarks/0.17.3--after-lorem.txt:1]: Ideally I'd have benchmarks comparing before and after in the same benchmark suite, but that's a real schlep to set up.

**Pull request merged**